src/notify-middleware.js

```diff
@@ -19,7 +19,7 @@ const notifyAndDispatch = store => next => action => {
       setTimeout(() => {
         window.scroll(0, 0);
       }, 100);
-    } else if (action.payload && action.payload.message) {
+    } else if (action.payload && action.payload.message && !action.payload.message.includes("letter")) {
```

Comment:
Tell me more about removing the payload here. Why do we want to remove it if it was intended to come from the backend?

Reply:
I am a bit unclear to what is going on here but it looks like a very specific exception to a rule and that should probably be fixed somewhere else. What should we do in the backend to not have to do the above?

Reply:
I added the specific exception to disable the notification when page is loaded and to enable that it will show correct notification error message after adding the confirmation code. 
However, when GET_LETTER_PROOFING_PROOFING_SUCCESS(triggered when page is loaded) will cause action.payload && action.payload.message that will show the notification error message.
this error message will cover the other notification error messages, for example when user enters incorrect input value.
Alternatively, is it possible to remove this notification message from backend, since the information is showed in the vetting button now?

Reply:
The payload with type GET_LETTER_PROOFING_PROOFING_SUCCESS is the one containing the current state of the users letter proofing. I thought this was the one that provided the data to be shown in the button?

Reply:
Yes, If its possible to remove the message from backend, I can remove the specific rules in frontend, since the new button will display this information...

Reply:
Yes, so we should release this PR and then remove the message from backend responses, then remove this special hack.

Reply:
:+1: